README.md

```diff
@@ -5,3 +5,34 @@
 [![Codecov](https://codecov.io/gh/dgan181/QuDiffEq.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/dgan181/QuDiffEq.jl)
 
 Quantum algorithms for solving differential equations.
+
+This project is part of Julia Season of Contribution 2019.
+It is still in the initial development phase.
+
+## Getting Started
+
+To add package:
+
+```
+pkg> add https://github.com/QuantumBFS/QuDiffEq.jl.git
+```
+
+## Features
+
+- Quantum Algorithms for Linear Differential Equations
+  - Based on Taylor Expansion (ref : arxiv.org/abs/1807.04553)
+  - Based on HHL (ref : arxiv.org/abs/1010.2745v2)
+
+
+## Built With
+
+* [Yao](https://github.com/QuantumBFS/Yao.jl) - A framework for Quantum Algorithm Design
+* [QuAlgorithmZoo](https://github.com/QuantumBFS/QuAlgorithmZoo.jl) - A repository for Quantum Algorithms
```

Comment:
Do we still have this as dependency?

Reply:
No. I added that because I have code from the repository (HHL, QFT and PE blocks).